localstripe/resources.py

```diff
@@ -1527,6 +1530,8 @@ def __init__(self, amount=None, currency=None, customer=None,
                 assert (payment_method.startswith('pm_') or
                         payment_method.startswith('src_') or
                         payment_method.startswith('card_'))
+            assert type(off_session) is bool
+            assert type(confirm) is bool
```

Comment:
You need to
```suggestion
            if confirm is not None:
                assert type(confirm) is bool
            if off_session is not None:
                assert type(off_session) is bool
```
Otherwise current requests (without passing `confirm`) won't work anymore, and many users will complain.